app/scripts/views/header.coffee

```diff
@@ -39,6 +41,9 @@ module.exports = class HeaderView extends View
     # Update date in all posts/comments on the page
     setInterval RefreshDateView::tick, 60000
 
+    # Check anonymous status every 500ms and indicate it with icon
+    setInterval @RefreshAnonymousStatus, 500
```

Comment:
You added that in order to update anonymous status if multiple tabs with meow are opened? That's ok, but are you sure this need to be called twice every second? Maybe 2000-5000ms would be more appropriate?

Also, you don't need to call it if user is not logged.